doc/book/admin/access_control.rst

```diff
@@ -1,270 +1,865 @@
-      box.schema.user.grant('U', 'write', 'space', '_priv')
-    -- So that 'U' can create roles:
-      box.schema.user.grant('U','create','role')
-      box.schema.user.grant('U', 'read,write', 'space', '_user')
-      box.schema.user.grant('U', 'write', 'space', '_priv')
-    -- So that 'U' can create sequence generators:
-      box.schema.user.grant('U','create','sequence')
-      box.schema.user.grant('U', 'read,write', 'space', '_sequence')
-    -- So that 'U' can create functions:
-      box.schema.user.grant('U','create','function')
-      box.schema.user.grant('U','read,write','space','_func')
-    -- So that 'U' can create any object of any type
-      box.schema.user.grant('U','read,write,create','universe')
-    -- So that 'U' can grant access on objects that 'U' created
-      box.schema.user.grant('U','write','space','_priv')
-    -- So that 'U' can select or get from a space named 'T'
-      box.schema.user.grant('U','read','space','T')
-    -- So that 'U' can update or insert or delete or truncate a space named 'T'
-      box.schema.user.grant('U','write','space','T')
-    -- So that 'U' can execute a function named 'F'
-      box.schema.user.grant('U','execute','function','F')
-    -- So that 'U' can use the "S:next()" function with a sequence named S
-      box.schema.user.grant('U','read,write','sequence','S')
-    -- So that 'U' can use the "S:set()" or "S:reset() function with a sequence named S
-      box.schema.user.grant('U','write','sequence','S')
-    -- So that 'U' can drop a sequence (assuming 'U' did not create it)
-      box.schema.user.grant('U','drop','sequence')
-      box.schema.user.grant('U','write','space','_sequence_data')
-      box.schema.user.grant('U','write','space','_sequence')
-    -- So that 'U' can drop a function (assuming 'U' did not create it)
-      box.schema.user.grant('U','drop','function')
-      box.schema.user.grant('U','write','space','_func')
-    -- So that 'U' can drop a space that has some associated objects
-      box.schema.user.grant('U','create,drop','space')
-      box.schema.user.grant('U','write','space','_schema')
-      box.schema.user.grant('U','write','space','_space')
-      box.schema.user.grant('U','write','space','_space_sequence')
-      box.schema.user.grant('U','read','space','_trigger')
-      box.schema.user.grant('U','read','space','_fk_constraint')
-      box.schema.user.grant('U','read','space','_ck_constraint')
-      box.schema.user.grant('U','read','space','_func_index')
-    -- So that 'U' can drop any space (ignore if the privilege exists already)
-      box.schema.user.grant('U','drop','space',nil,{if_not_exists=true})
-
-**Example for creating users and objects then granting privileges**
-
-Here a Lua function is created that will be executed under the user ID of its
+    box.schema.user.grant(user-name, privileges, object-type, object-name[, {options}])
+
+*   ``user-name``: the name of the user that gets the specified privileges.
```

Comment:
Hmm.. In this case we need to fix this in all reference pages related to privileges (because I just copy-pasted this from the `box.schema.user.grant` page).